<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoleManager</class>
 <widget class="QWidget" name="RoleManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="leftPanel" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, stop:0 rgba(248, 230, 255), stop:1 rgb(85, 255, 255));
    border: 1px solid #ccc;
    color: black;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="10" column="0">
          <widget class="QTextEdit" name="descriptionEdit"/>
         </item>
         <item row="10" column="1">
          <widget class="QTextEdit" name="styleEdit">
           <property name="maximumSize">
            <size>
             <width>570</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLabel" name="styleLabel">
           <property name="text">
            <string>角色提示词</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="descriptionLabel">
           <property name="text">
            <string>角色描述：</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>角色名称：</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLineEdit" name="nameEdit">
           <property name="minimumSize">
            <size>
             <width>237</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="rightPanel" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QListWidget" name="roleList">
         <property name="styleSheet">
          <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, stop:0 rgba(248, 230, 255), stop:1 rgb(85, 255, 255));
    border: 1px solid #ccc;
    color: black;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="styleSheet">
          <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, stop:0 rgba(248, 230, 255), stop:1 rgb(85, 255, 255));
    border: 1px solid #ccc;
    color: black;</string>
         </property>
         <property name="text">
          <string>添加角色</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeButton">
         <property name="styleSheet">
          <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, stop:0 rgba(248, 230, 255), stop:1 rgb(85, 255, 255));
    border: 1px solid #ccc;
    color: black;</string>
         </property>
         <property name="text">
          <string>删除角色</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
